src/lib.rs

```diff
@@ -21,11 +21,11 @@
 // validation of a field while handling the case of the field not existing.
 #![allow(clippy::blocks_in_if_conditions)]
 
-#[cfg(all(feature = "ck3", feature = "vic3"))]
-compile_error!("features \"ck3\" and \"vic3\" cannot be enabled at the same time");
+#[cfg(all(feature = "ck3", feature = "vic3", feature = "imperator"))]
```

Comment:
We'll have to make this more complicated and check for any two or more feature flags :(

Unless I fix it so they can be on at the same time :)